docs/source/quickstart.mdx

```diff
@@ -0,0 +1,545 @@
+              }
+            }
+          ]
+        }
+      }
+    }
+  }
+  ```
+
+    $600.25! That must be some t-shirt!
+
+  </ExpansionPanel>
+
+Excellent! You've used one request to get information from across different services, including the `users`, `products`, and `inventory` subgraphs.
+In the last part of this step, you'll learn more about how the router accomplished this.
+
+#### Inspect query plans
+
+The `GetCart` operation is powerful because it gathers data across various subgraphs: `users`, `products`, `inventory`, and more.
+Sandbox displays an operation's **Query Plan** to help you understand how the router intelligently orchestrates subrequests to these subgraphs.
+
+A query plan is a blueprint for dividing a single incoming operation into one or more operations that are each resolvable by a single subgraph.
+You can toggle to view the query plan instead of the response by clicking **Response** and selecting **Query Plan**.
+
+<img
+  src="./images/quickstart/query-plan.jpg"
+  className="screenshot"
+  width="700"
+  alt="Apollo Sandbox shows a query plan diagram for an operation."
+/>
+
+Query plans are a powerful tool for understanding operations, whether they're run locally or from client applications.
+
+In the last step of this tutorial, you'll publish your locally running subgraph schemas to GraphOS Studio to learn more about your supergraph.
+
+### Step 3. Publish your subgraphs
+
+To create a supergraph, you individually publish each subgraph schema to GraphOS. 
+Whenever you publish a subgraph schema, GraphOS attempts to compose all the latest subgraph schemas into a single supergraph schema.
+If this composition succeeds, GraphOS updates your router with the result.
+
+In this step, you'll create an empty supergraph in GraphOS Studio and then publish subgraph schemas to it using Rover.
+
+#### Create supergraph
+
+1. Go to [studio.apollographql.com](https://studio.apollographql.com/?referrer=docs-content) and click **+ Create New Graph**.
+2. Enter a **Graph title**, for example `Federation Quickstart`. Leave the **Graph Architecture** as `Supergraph (Default)` and **Visible to Team Members** as `On`. Click **Next**.
+3. A **Publish your schema using Federation** dialog appears. Copy the example code block to a local text file.
+
+    <img
+      src="./images/quickstart/publish-your-schema.jpg"
+      className="screenshot"
+      width="500"
+      alt="A dialog from GraphOS Studio displays a hidden API key and graph ref."
+    />
+
+    - The protected value that appears after `APOLLO_KEY=` and beginning with `service:` is your graph API key.
+    - The value that appears after `rover subgraph publish` is your graph ref. By default, it ends with `@current`.
+
+    <Caution>
+
+      API keys are secret credentials. Never share them outside your organization or commit them to version control. Delete and replace API keys that you believe are compromised.
```

Comment:
when you copy manually the first two lines (as I did, bc I dindn't wanted the last thre and the copy btn would have given me the whole thing, then you get the dots copied over instead of the key, unless click the reveal btn and do it again). Maybe we should be more specific in the guidance?  
<img width="731" alt="Screenshot 2024-06-21 at 16 20 25" src="https://github.com/apollographql/federation/assets/1368445/bda538d9-5895-4179-bc63-ac1df9371876">

Reply:
https://github.com/apollographql/federation/pull/3037/commits/5fc4f6e6c127cc23544f1452ab5c0381b6c865da